lib/ui/platform_dispatcher.dart

```diff
@@ -801,11 +801,46 @@ class PlatformDispatcher {
   ///
   ///  * [SchedulerBinding], the Flutter framework class which manages the
   ///    scheduling of frames.
+  ///  * [scheduleWarmUpFrame], which should only be used to schedule warm up
+  ///    frames.
   void scheduleFrame() => _scheduleFrame();
 
   @Native<Void Function()>(symbol: 'PlatformConfigurationNativeApi::ScheduleFrame')
   external static void _scheduleFrame();
 
+  /// Schedule a frame to run as soon as possible, rather than waiting for the
+  /// engine to request a frame in response to a system "Vsync" signal.
+  ///
+  /// This method is used during application startup so that the first frame
+  /// (which is likely to be quite expensive) can start a few extra milliseconds
+  /// earlier. Using it in other situations might lead to unintended results,
+  /// such as screen tearing. Depending on platforms and situations, the warm up
+  /// frame might or might not be actually rendered onto the screen.
```

Comment:
Do we also want to use this when rendering the first frame in a new `FlutterView`? I imagine we'd want the new view to be rendered as fast as possible too.

Reply:
That's a good point, although I imagine this would require more work. Can I open an issue to leave the optimization for later?

Reply:
I don't think this optimization works for individual views. Because of the unified widget tree we can only produce frames for all views at once and cannot target individual views. This optimization only works for the app as a whole.

Reply:
Nope, I was just hypothesizing. If we don't need it, then we don't need it. My gut feeling is that we don't need any of this on the web. So I'm actually a little sad that this API even exists.